### PR TITLE
Bump dataframe/1.15.0

### PR DESCRIPTION
**Pull request by theirix**
Specify library name and version:  **dataframe/1.15.0**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by Croydon — DISMISSED** [review]

**Review by prince-chrismc — APPROVED** [review]

**Review by dmn-star — DISMISSED** [review]

**Review by hosseinmoein — APPROVED** [review]

**Comment by hosseinmoein** [comment]
I closed the other conflicting PR

**Comment by hosseinmoein** [comment]
Is there any resolution as to why this is not building?
Thanks

**Comment by hosseinmoein** [comment]
@Croydon , @prince-chrismc ,
Is there anything we can do to move this forward?
Thanks

File: recipes/dataframe/all/conanfile.py [hunk 1]
```diff
@@ -79,11 +78,11 @@ def configure(self):
             )
             return
 
-        # Exclude compilers not supported by cpp-taskflow
+        # Exclude compilers not supported
         if compiler_version < minimal_version[compiler]:
             raise ConanInvalidConfiguration(
                 "{} requires a compiler that supports at least C++17. {} {} is not supported.".format(
-                    self.name, compiler, Version(self.settings.compiler.version.value)
+                    self.name, compiler, tools.Version(self.settings.compiler.version.value)
```
Comment (prince-chrismc) [inline recipes/dataframe/all/conanfile.py:85]:
```suggestion
                    self.name, compiler, tools.Version(self.settings.compiler.version)
```

**Review by prince-chrismc — CHANGES_REQUESTED** [review]

**Comment by prince-chrismc** [comment]
You'll need to restart it again

**Comment by theirix** [comment]
> You'll need to restart it again

didn't help

**Comment by prince-chrismc** [comment]
Status is in progress... it's doing something  now

**Review by hosseinmoein — APPROVED** [review]

**Review by prince-chrismc — APPROVED** [review]

**Review by SpaceIm — APPROVED** [review]

**Review by mathbunnyru — APPROVED** [review]

**Comment by hosseinmoein** [comment]
@prince-chrismc , @SpaceIm , @mathbunnyru 
Is this mergeable now?
Thanks,

**Comment by prince-chrismc** [comment]
#2857 Has some details about what's required for merging, but we are missing one for the CCI team...

Please be patient they are doing their best to keep up with all the PRs we get

**Comment by mathbunnyru** [comment]
Sorry, I'm not official community reviewer, just trying to help the CCI team when I can.

**Review by SSE4 — APPROVED** [review]

**Pull request merged**